### DISCUSSION TITLE
Email when task successful finished

### DISCUSSION THREAD

**Discussion by chevcheli0s** [ptarmiganlabs/butler#1207] [Ideas]
Hi! Thanks for the great project!
Maybe you can implement a simple version of Qlik Alerting.
When a task completes with a status of "success" and it have a custom property (with email addresses) with a name that matches in the butler configuration (like in the case of a failed tasks), the butler will send a success email from a specific template.

**Comment by mountaindude** [comment]
I am doing a major overhaul of the alert messages, and will add support for this ticket.

So far a bunch of new template variables (various app and task metadata) have made available to Teams, Slack and email messages for failed and stopped reload tasks. 

Same set of metadata will be available for successful reload tasks. 

Recipients of successful-reload-task-emails will be determined using same logic as failed/stopped tasks, except that app owners will not receive successful reload alerts (this can be added if needed, I just don't see the need for it right now).
I.e.

- Email addresses specified via custom property on the reload task
- Email addresses specified in the Butler config file, if a certain custom property is set to a value that's also defined in the config file.

**Comment by mountaindude** [comment]
Delivered in 13.0.

More info here: https://butler.ptarmiganlabs.com/docs/concepts/successful-reloads/
